### ISSUE TITLE
Embulk generate executable document.

### ISSUE THREAD

**Issue by mitoma** [embulk/embulk#308]
Example
```sh
$ embulk new ruby-input test
2015-09-22 23:57:20.379 +0900: Embulk v0.7.4
Creating embulk-input-test/
---snip---

$ cd embulk-input-test/
$ ls -l
total 24
total 40
drwxrwxr-x  3 mitoma mitoma 4096  9月 22 23:57 .
drwxrwxr-x 28 mitoma mitoma 4096  9月 22 23:57 ..
-rw-rw-r--  1 mitoma mitoma   39  9月 22 23:57 .gitignore
-rwxrwxr-t  1 mitoma mitoma   14  9月 22 23:57 .ruby-version <<-- executable
-rwxrwxr-t  1 mitoma mitoma   39  9月 22 23:57 Gemfile       <<-- executable
-rwxrwxr-t  1 mitoma mitoma 1037  9月 22 23:57 LICENSE.txt   <<-- executable
-rw-rw-r--  1 mitoma mitoma  511  9月 22 23:57 README.md
-rwxrwxr-t  1 mitoma mitoma   50  9月 22 23:57 Rakefile      <<-- executable
-rw-rw-r--  1 mitoma mitoma  841  9月 22 23:57 embulk-input-test.gemspec
drwxrwxr-x  3 mitoma mitoma 4096  9月 22 23:57 lib
```

**Comment by frsyuki** [comment]
do you mean that permission should be 0644?

**Comment by mitoma** [comment]
umask is ubuntu's default.

```
$ umask
0002
```

**Comment by frsyuki** [comment]
Fixed at 9279da5.
